### ISSUE TITLE
Dark Mode makes Xaero's Minimap darker [forge 1.19.2]

### ISSUE THREAD

**Issue by Yuda131** [Buuz135/DarkModeEverywhere#46]
Using Dark Mode Everywhere mod also darkens the Xaero's Minimap mod. Is there any way to stop it from affecting Xaero's Minimap?

**Comment by Lordfirespeed** [comment]
Yes, there is, but it's a bit contrived - it involves the shader method blacklist in the mod config.

**Comment by Lordfirespeed** [comment]
it's not trivial to explain 😓 take a look at the `README.md` file and see if you can crack it

**Comment by Gotolei** [comment]
For forge 1.20.1, adding these to the shader blacklist seems to do the trick:
`xaero.common.minimap.render.MinimapRenderer:renderMinimap`
`xaero.common.graphics.GuiHelper:innerBlit`
